src/pages/NotFound/NotFoundPage.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button, Card, CardActions, CardMedia, Stack } from "@mui/material";
+import React from "react";
+import { LayoutContainer } from "../../shared/LayoutContainer";
+import EdugatorNotFound from "../../assets/EdugatorNotFound.png";
+import { useHistory } from "react-router-dom";
+import { LocalStorage } from "../../app/common/LocalStorage";
+import { Routes } from "../../shared/Routes.constants";
+
+export function NotFoundPage() {
+  const history = useHistory();
+  return (
+    <LayoutContainer pageTitle={"404: Page Not Found"}>
+      <Stack>
+        <Card>
+          <CardActions>
+            <Button
+              variant="contained"
+              color="primary"
+              onClick={() => history.push(Routes.Landing)}
+            >
+              Homepage
+            </Button>
+            {LocalStorage.getToken() && (
+              <Button
+                variant="contained"
+                color="primary"
+                onClick={() => history.push(Routes.Modules)}
+              >
+                Modules
+              </Button>
+            )}
+            {!!!LocalStorage.getToken() &&
+              history.location.pathname.indexOf("admin") !== -1 && (
+                <Button
+                  variant="contained"
+                  color="primary"
+                  onClick={() => history.push(Routes.Login)}
+                >
+                  Admin Login
+                </Button>
```

Comment:
Giving unauthenticated people a button to go to the admin login is weird. 

Even if guessing where the admin login page is easy.

Reply:
I think it is fine if they are at a non-existent /admin route. The /admin/login route is easy to guess and you can also find it on Github.

Reply:
I think I am good either way.  I agree that whichever we choose, we can't really hide the admin login page well